docs/package-lock.json

```diff
@@ -14,6 +14,7 @@
         "clsx": "^1.2.1",
         "graphiql": "^2.4.7",
         "graphql": "^16.6.0",
+        "iframe-resizer-react": "^1.1.0",
```

Comment:
We should remove this file and use yarn.lock exclusively not npm

Reply:
Maybe add package-lock.json to gitignore by the way since yarn is used across the repo?

Reply:
i was removing and caching it while i solved the issue with yarn on my machine (solved it now) but i'll add it to gitignore too